web/src/beta/components/Text/index.tsx

```diff
@@ -57,7 +59,8 @@ const Text: React.FC<Props> = ({
         ...otherProperties,
         color: customColor ? undefined : color || defaultColor,
       }}
-      onClick={onClick}>
+      onClick={onClick}
+      onDoubleClick={onDoubleClick}>
```

Comment:
By just adding this onDoubleClick, without setting a timeout or someway to subvert it, you will get onClick fired too, which isn't desirable.
Can you do away with the double click for now? I am using SHIFT+click elsewhere, knowing that we need to update to the double click in the future. We just need to remember where this double click is needed so we can update it later with the rest

Reply:
I'll do it in a seperate PR.